content/contributing/documentation/rst_guidelines.rst

```diff
@@ -1,162 +1,1383 @@
 
-#. Relative:
+.. _contributing-rst-hyperlinks-guidelines:
 
-   - ``../prices.html``
-   - ``variants.html``
+Hyperlinks
+==========
 
-The relative links are clearly superior in terms of readability and stability: the references
-survive version updates, folder name changes and file tree restructurations.
+.. _contributing-rst-relative-links:
 
-.. _contributing/line-length-limit:
+Internal URLs: relative links
+-----------------------------
 
-Start a new line before the 100th character
-===========================================
+If you need to reference an :ref:`internal documentation page <contributing-rst-doc-page-hyperlinks>`
+or a :ref:`file <contributing-rst-file>` that is not located in the same directory as
+your current page, always use *relative file paths* instead of *absolute file paths*. This ensures
+that links remain valid even with version updates, folder name changes, and directory structure
+reorganizations.
 
-In RST, it is possible to break a line without forcing a line break on the rendered HTML. Make use
-of this feature to write **lines of maximum 100 characters**. A line break in a sentence results in
-an additional whitespace in HTML. That means that you do not need to leave a trailing whitespace at
-the end of a line to separate words.
+An absolute file path indicates the target's location from the root directory. A relative file path
+uses smart notations (such as ``../`` that redirects to the parent folder) to indicate the target's
+location *relative* to that of the source document.
 
-.. tip::
-   You can safely break a line around the separators (``-->``) of ``menuselection`` markups and
-   anywhere in a hyperlink reference. For the ``doc``, ``ref`` and ``download`` markups, this is
-   only true for the label part of the reference.
+.. example::
+   Given the following source file tree:
+   ::
 
-Example: Line breaks within markups
------------------------------------
+     documentation
+     ├── content
+     │   └── applications
+     │   │   └── sales
+     │   │   │   └── sales
+     │   │   │   │   └── products_prices
+     │   │   │   │   │   └── products
+     │   │   │   │   │   │   └── import.rst
+     │   │   │   │   │   │   └── variants.rst
+     │   │   │   │   │   └── prices.rst
 
-.. code-block:: rst
+   A reference to the :file:`prices.rst` and :file:`variants.rst` could be made from
+   :file:`import.rst` as follows:
 
-   To register your seller account in Odoo, go to :menuselection:`Sales --> Configuration --> Settings
-   --> Amazon Connector --> Amazon Accounts` and click on :guilabel:`CREATE`. You can find the **Seller
-   ID** under the link :guilabel:`Your Merchant Token`.
+   #. Absolute:
 
-Be consistent with indentation
-==============================
+      .. code-block:: rst
+
+         - :doc:`documentation/content/applications/sales/sales/products_prices/prices.rst`
+         - :doc:`documentation/content/applications/sales/sales/products_prices/products/variants.rst`
```

Comment:
```suggestion
         - :doc:`/applications/sales/sales/products_prices/prices`
         - :doc:`/applications/sales/sales/products_prices/products/variants`
```
"Absolute" links with the `:doc:` markup start from `/applications` and don't include the `.rst `extension

Reply:
Yeah I know, I was just trying to find a "universal" example but I guess that's not possible without confusing people (or being inaccurate in one way or another), you're right ...